app/models/user.rb

```diff
@@ -3,6 +3,7 @@
 # Table name: users
 #
 #  id                  :integer          not null, primary key
+# FIXME: rename to username
```

Comment:
I overlooked this before writing a patch for User.username. But I think `username` is preferable to to `mw_username`. Although we do make some queries against username where we assume that it's the same one on Wikipedia, most of the uses throughout the app are just invoking the username that should be shown for a user.

Unlike the revision and page ids, this *is* a global, cross-wiki username. If you can log in with OAuth, then you have a unified account and the same name across Wikimedia wikis (right?).